src/main/java/core/basesyntax/service/impl/ConvertServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.service.ConvertService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConvertServiceImpl implements ConvertService {
+    private static final String SEPARATOR = ",";
+    private static final int INDEX_BY_TRANSACTION = 0;
+    private static final int INDEX_BY_FRUIT = 1;
+    private static final int INDEX_BY_AMOUNT = 2;
+
+    @Override
+    public List<FruitTransaction> proccesing(List<String> strings) {
```

Comment:
```suggestion
    public List<FruitTransaction> convertText(List<String> strings) {
```
I would suggest something like "convertText" as method name